README.md

```diff
@@ -244,7 +244,7 @@ We gladly credit all contributors.
     and other financial contexts, but when we talk about equity compensation, it is meant more
     informally.
 - **Why give equity compensation?**
-  The purpose of equity compensation is twofold:
+  The purpose of equity compensation is twofold.
```

Comment:
Curious why this change? https://github.com/feynmanlabs/editorial-quality#colons